input/pagecontent/comparison.md

```diff
@@ -1,1206 +1,499 @@
+- Media observation (Results: laboratory, media)
+- Medication Statement (IPS)
+- Observation - Pregnancy: EDD
+- Observation - Pregnancy: outcome
+- Observation - Pregnancy: status
+- Observation - SH: alcohol use
+- Observation Results: radiology (IPS)
+- Specimen (IPS)
+
+#### US Core
+[US Core Implementation Guide 7.0.0](https://hl7.org/fhir/us/core/2024Jan/) serves as the foundation for FHIR implementation guides in the US Realm. It establishes the minimum constraints on FHIR resources to define US Core Profiles, specifying the required elements, extensions, vocabularies, and value sets, as well as their usage. It also outlines the minimum FHIR RESTful interactions needed to access patient data for each US Core Profile.
+
+AU Core was developed with consideration of the REST API and profiles in the US Core Implementation Guide, ensuring compatibility where possible. To achieve compliance with both AU Core and US Core, additional requirements may need to be met.T hese requirements are outlined below.
+
+<table border="1" style="width: 100%; margin: auto; border-collapse: collapse;">
+    <thead>
+        <tr>
+            <th style="width: 25%;">AU Core</th>
+            <th style="width: 25%;">US Core 7.0.0</th>
+            <th style="width: 25%;">Element</th>
+            <th style="width: 25%;">Additional requirements</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td rowspan="2" style="width: 25%;"><a href="StructureDefinition-au-core-condition.html">AU Core Condition</a></td>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-condition-problems-health-concerns.html">US Core Condition Problems and Health Concerns</a></td>
+            <td style="width: 25%;">Category.code</td>
+            <td style="width: 25%;">US Core requires a value from <a href="https://hl7.org/fhir/us/core/STU7/ValueSet-us-core-problem-or-health-concern.html">US Core Problem or Health Concern</a> value set.</td>
+        </tr>
+        <tr>
+            <td style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-condition-encounter-diagnosis.html">US Core Condition Encounter Diagnosis Profile</a></td>
+            <td style="width: 25%;">Category.code</td>
+            <td style="width: 25%;">US Core requires 'encounter-diagnosis' value.</td>
+        </tr>
+        <tr>
+            <td rowspan="2" style="width: 25%;"><a href="StructureDefinition-au-core-allergyintolerance.html">AU Core Diagnostic Result Observation</a></td>
+            <td rowspan="2" style="width: 25%;"><a href="https://hl7.org/fhir/us/core/STU7/StructureDefinition-us-core-observation-clinical-result.html">US Core Observation Clinical Result</a></td>
+            <td style="width: 25%;">Observation.code</td>
+            <td style="width: 25%;">US Core has an extensible binding to <a href="https://hl7.org/fhir/R4/valueset-observation-codes.html">LOINC Codes</a>.</td>
```

Comment:
Rephrase all the 'extensible' comments to be in the format of "US Core extensible binding to <>."

Reply:
Done

Reply:
Partially done. change "extensibly binds to" to be "extensible binding to"

Reply:
Can't find this, please let me where.

Reply:
I suspect it was a commit that wasn't pushed last time. It is not there now.